deepchem/models/torch_models/tests/test_progressive_multitask.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import deepchem as dc
+
+import pytest
+
+try:
+    import torch
+    import torch.nn as nn
+
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+
+
+@pytest.mark.torch
+def test_construction():
+    """
+    Test that PrgressiveMultiTask Model can be constructed without crash.
+    """
+
+    model = dc.models.torch_models.ProgressiveMultitask(
+        n_tasks=1,
+        n_features=100,
+        layer_sizes=[128, 256],
+        alpha_init_stddevs=[0.08],
+        weight_init_stddevs=0.02,
+        bias_init_consts=1.0,
+        activation_fns=nn.ReLU,
+        dropouts=[0.2],
+        n_outputs=2,
+    )
+
+    assert model is not None
+
+
+@pytest.mark.torch
+def test_forward():
+    """
+    Test that the forward pass of ProgressiveMultiTask Model can be executed without crash
+    and that the output has the correct value.
+    """
+
+    n_tasks = 2
+    n_samples = 20
+    n_features = 12
+    np.random.seed(123)
+
+    X = np.random.rand(n_samples, n_features)
```

Comment:
change this to a permanent input file instead of randomly generating the input

Reply:
Done